torch/csrc/jit/codegen/cuda/lower_warp_reduce.cpp

```diff
@@ -84,7 +84,7 @@ class EliminateDeadBroadcastAndAllocate {
         if (allocate->memoryType() == MemoryType::Local) {
           if (auto tv = dynamic_cast<TensorView*>(allocate->buffer())) {
             // We know only tvs that we'd want to consider are broadcast outputs
-            if (tv->definition()->isA<BroadcastOp>()) {
+            if (tv->definition() && tv->definition()->isA<BroadcastOp>()) {
```

Comment:
Curious when `tv` can be null? We are allocating a buffer for the tv here, but that tv doesn't have a definition. I can't think of when that could happen.